### ISSUE TITLE
v0.43.0 breaks viewing of *.glb files

### ISSUE THREAD

**Issue by qqmyers** [Smithsonian/dpo-voyager#297]
I've just been working to display glb files such as https://dataverse.harvard.edu/file.xhtml?fileId=6931827 within [Dataverse](https://dataverse.org/). 

We're generating HTML like 
`<voyager-explorer 
  resourceroot="https://3d-api.si.edu/resources/" 
  root="http://ourhost.org/api/v1/access/datafile/" 
  model="ClunyCapital06.glb">
</voyager-explorer>` 

and then loading the voyager-explorer.min.js script. 

Using https://3d-api.si.edu/resources/js/v0.42.1/voyager-explorer.min.js (and several earlier versions), this works.
With https://3d-api.si.edu/resources/js/v0.43.0/voyager-explorer.min.js, we see the following error in the display:

![image](https://github.com/user-attachments/assets/8639f84e-d664-4a62-aed6-1f38010f6c19) and nothing displays.

I'm not familiar enough with the format or the Voyager app to help debug this further, but I can point to other glb examples that fail the same way and can probably get answers about how these files were created if that helps. (Am I just using the root/model attributes incorrectly?) 

I've also confirmed that we see the same problem using a static HTML file with just voyager-explorer element above followed by `<script src="https://3d-api.si.edu/resources/js/v0.43.0/voyager-explorer.min.js"></script>`

Thanks for any help and apologies if this is an issue on our end and not a bug.

Also FWIW, we found that the glb file was being loaded via https://github.com/Smithsonian/dpo-voyager/blob/ebbda0b08e7ab3ec12051e59230d1a1bd56b01a2/source/client/io/FileReader.ts#L63 and our code was originally responding with a 406 because the mimetype is model/gltf-binary and not text/plain as was being requested. (It was the same in v0.42.1, so I assume it's not related, and we've worked around it, but I mention it in case it means we're not loading the file correctly)).

**Comment by gjcope** [comment]
Hi @qqmyers , thanks for the bug report and awesome to hear about the inclusion in Dataverse!

This is definitely a bug on our side introduced by an unnoticed type change. It's been fixed in [160a19f](https://github.com/Smithsonian/dpo-voyager/commit/160a19f2d2ede3ee606a388528123e521d8e0069) and will be included in the next release. As a temporary fix, you can manually set the quality parameter (ex: quality="High") to achieve the same thing.

FWIW I believe the model-only loading mechanism is not often used with Voyager as most of the viewer's value comes from content in the [Voyager scene file](https://smithsonian.github.io/dpo-voyager/document/example/) compared to other lower-overhead viewers. Are you planning to support Voyager scenes in Dataverse as well? That would be super cool - happy to start another thread on the that topic if it would be helpful.

**Comment by qqmyers** [comment]
@gjcope - thanks for the fix and work-around! I've just added it so we can use 0.43.0.

As for supporting scene files - in general we follow the Dataverse users and I'm not sure how many datasets with scene files there are so far. I did look into the technical aspects. One challenge for us is that we don't support accessing files by their original path/name (even relative path). We also support having per-file access restrictions and embargoes and direct access to files on S3 stores using signed URLs. All of that means that somewhere we'd have to manage changing the scene file's references to our download URLs, handle error conditions where someone has restricted/embargoed a file needed by the scene file, etc. Not impossible, but more work that just pointing Voyager at the glb file.
